packages/manager-components/src/components/region/region.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { screen } from '@testing-library/react';
+import { render } from '../../utils/test.provider';
+import Region, { RegionProps } from './region.component';
+import { DemoRegion, DemoDatacenter } from './region.stories';
+import translatedRegion from './translations/region/Messages_fr_FR.json';
+
+const renderComponent = (props: RegionProps) => {
+  return render(<Region {...props} />);
+};
+
+describe('Region component', () => {
+  it('renders region correctly for region key', async () => {
+    renderComponent({
+      ...DemoRegion.args,
+    });
+    const regionKey = DemoRegion.args.name
+      .replace(/-/g, '_')
+      ?.toLocaleLowerCase();
```

Comment:
**question:** Are you sure this test-case is passing?

Reply:
Its corrected now.